localisation/english/btr_origins_l_english.yml

```diff
@@ -18,4 +18,10 @@
+
+
+### Parnack Regime Origin ###
+
+btr_origin_parnkreg:0 "A Trail of Space Preys"
+btr_origin_parnkreg_desc:0 "As the apex predators on our world, we Yahg refused to kneel before the star creatures when they came to Parnack demanding allegiance to their Citadel. We only serve those who have proven to be the strongest, the fittest, the smartest ones in the pack. The star creatures shunned us, confined us to our world, but we have finally reached the stars on our own terms. Soon, we will dominate the entire galaxy by the virtue of our strength! They will have to bow before us, or die."
```

Comment:
I'm not really keen on the idea of the Yahg referring to the other species as "Star Creatures." They're supposed to be incredibly smart as well as strong, as referenced elsewhere, I see no reason they wouldn't just say "species" or whatever, let's get that revisited.